oml/utils/io.py

```diff
@@ -37,12 +37,15 @@ def calc_folder_hash(folder: Union[Path, str]) -> str:
     return folder_hash.hexdigest()
 
 
-def check_exists_and_validate_md5(path: Union[str, Path], md5: str) -> bool:
+def check_exists_and_validate_md5(path: Union[str, Path], md5: Optional[str] = None) -> bool:
```

Comment:
What is happening here?)

Looks weird in terms of naming and expected functionality. Like:
```python 
def compare_two(a, b=None):
    if not b:
        return True
```

Reply:
rolled back